lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -15,67 +13,42 @@ class FeatureCard extends React.Component {
   constructor(props) {
     super(props);
 
-    // These variables shouldn't force a state change & subsequent render
-    this.requests = [];
-    this.campaigns = [];
-    this.campaignsToExclude = setting('dsUser.activity.signups', []);
-    this.campaignsApiPage = 1;
-    this.apiClient = new ApiClient('v1');
-
     this.state = {
       actionDisabled: false,
       campaignIndex: -1,
       isSignedUp: false,
       isLoading: false,
     };
 
+    this.campaigns = [];
+    this.controller = new GalleryController();
```

Comment:
So I find it weird though that it's the Feature Card that is implementing the `GalleryController` then... particularly for the slide that will have 3 `FeatureCard`s, then each will have an implementation of the `GalleryController`? That seems incorrect. Ideally, something else (the slide?) is instantiating the `GalleryController` (which sounds more like it's a `CampaignsController`), and then if it wants 1 `FeatureCard` or 3 `FeatureCard`s, then it creates as many as needed and passes the info the `FeatureCard` would need to make each of them. Does this make sense?